docs/docsite/rst/intro_inventory.rst

```diff
@@ -37,7 +37,28 @@ The headings in brackets are group names, which are used in classifying systems
 and deciding what systems you are controlling at what times and for what purpose.
 
 It is ok to put systems in more than one group, for instance a server could be both a webserver and a dbserver.
-If you do, note that variables will come from all of the groups they are a member of. Variable precedence is detailed in a later chapter.
+If you do, note that variables for a system will come from all of the groups they are a member of, regardless of whether it was referred to directly by name (such as ``someserver.example.com``) or indirectly via one of the groups it is part of (such as ``webservers``). Variable precedence is detailed in a :ref:`later chapter <variable_precedence>`.
```

Comment:
Since it's confusing, probably a sign that the pronoun is insufficient: 
> If you do, note that variables for a system will come from all of the groups they are a member of, regardless of whether the system was referred to [...]